test/unit/on-attach.spec.js

```diff
@@ -900,18 +808,22 @@ describe('onAttach', function() {
     });
 
     it('should trigger onBeforeAttach and onAttach on each of its childViews a single time', function() {
-      expect(this.childView1.onBeforeAttach).to.have.been.calledOnce;
-      expect(this.childView1.onBeforeAttach).to.have.been.calledOn(this.childView1);
-      expect(this.childView1.onBeforeAttach).to.have.been.calledWith(this.childView1);
-      expect(this.childView1.onAttach).to.have.been.calledOnce;
-      expect(this.childView1.onAttach).to.have.been.calledOn(this.childView1);
-      expect(this.childView1.onAttach).to.have.been.calledWith(this.childView1);
-      expect(this.childView2.onBeforeAttach).to.have.been.calledOnce;
-      expect(this.childView2.onBeforeAttach).to.have.been.calledOn(this.childView2);
-      expect(this.childView2.onBeforeAttach).to.have.been.calledWith(this.childView2);
-      expect(this.childView2.onAttach).to.have.been.calledOnce;
-      expect(this.childView2.onAttach).to.have.been.calledOn(this.childView2);
-      expect(this.childView2.onAttach).to.have.been.calledWith(this.childView2);
+      expect(this.childView1.onBeforeAttach)
+        .and.to.have.been.calledOnce
+        .and.to.have.been.calledOn(this.childView1)
+        .and.to.have.been.calledWith(this.childView1);
+      expect(this.childView1.onAttach)
+        .and.to.have.been.calledOnce
+        .and.to.have.been.calledOn(this.childView1)
+        .and.to.have.been.calledWith(this.childView1);
+      expect(this.childView2.onBeforeAttach)
+        .and.to.have.been.calledOnce
+        .and.to.have.been.calledOn(this.childView2)
+        .and.to.have.been.calledWith(this.childView2);
+      expect(this.childView2.onAttach)
+        .and.to.have.been.calledOnce
+        .and.to.have.been.calledOn(this.childView2)
+        .and.to.have.been.calledWith(this.childView2);
```

Comment:
I love this style of cleanup. Tons of improvements here @thejameskyle would be proud